scripts/merge_local_staging.py

```diff
@@ -21,27 +21,36 @@
 import sys
 
 
-def copy_and_append_index(target_directory, dir_path):
-    sub_dir_name = os.path.basename(os.listdir(dir_path)[0])
-    sub_dir_path = os.path.join(dir_path, sub_dir_name)
+def copy_and_append_index(target_directory, staging_directory):
```

Comment:
This script is too complex to understand. Either

1. Write comments and print message, add a method doc string to describe what it does.
2. Revert to the Netty script. I think it can handle a new staging variant and we actually need not to modify it.

Reply:
> too complex to understand

For example,

1. check dir or file multiple times, why?
2. manually concat filename, distinguish files and dirs. I'm afraid that it's brittle and doesn't work better than a bash script.

Reply:
A Pythonic way is using `pathlib` and logging each mutations.

Use `os` is almost the same as a bash script.

Reply:
Refactored